origin-growth/src/util/fixtures.js

```diff
@@ -19,15 +19,44 @@ function tokenNaturalUnits(x) {
 }
 
 const rule = {
-  numLevels: 3,
+  numLevels: 3
+  ,
   levels: {
     0: {
       rules: [
         {
-          id: 'PreRequisite',
+          id: 'ProfilePublished',
+          class: 'SingleEvent',
+          config: {
+            eventType: 'ProfilePublished',
+            reward: null,
+            nextLevelCondition: false,
+            visible: true
+          },
+        },
+        {
+          id: 'EmailAttestation',
+          class: 'SingleEvent',
+          config: {
+            eventType: 'EmailAttestationPublished',
+            reward: {
+              amount: tokenNaturalUnits(10),
```

Comment:
In the [spec](https://docs.google.com/document/d/1DJ00rl1MYSxkH5UMjVD0V-kzWU2uyRnAGA8x2YPqQdE/edit), it does not say email attestation will be rewarded - it is just used to qualify user for next level.

Reply:
good catch, changing to null